frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -12,7 +12,7 @@ import {
 import { ProjectSectionID } from './types';
 
 type DetailsSectionProps = {
-  id: ProjectSectionID;
```

Comment:
same comment as this PR

https://github.com/opendatahub-io/odh-dashboard/pull/983#discussion_r1126441396